### ISSUE TITLE
Name property missing for the padding text box.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#609]
**Describe the bug**

**Steps to reproduce the bug**

1. Turn on Narrator
2. Launch XAML Controls Gallery app.
3. Navigate to "Settings" and activate it.
4. Navigate to "Screenshot mode" toggle switch under 'Screenshots' header and  turn on the toggle switch.
5. Now navigate to "All Controls -> Acrylic(or any other list item in all controls) and activate it.
6. Navigate to 'padding' edit box and observe the narrator reading.

**Expected behavior**

Name property should be present for the edit box and should announce as 'Padding edit box'.

**Actual behavior**

There is no name property for the edit box and narrator announces as 'edit box'

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/102147889-58863c80-3e20-11eb-9b7b-f46af8683178.png)

**Version Info**
- [x] App downloaded from Store 
    - If so, which version? (Found on the Settings page of the app.) - 1.2.17.0
- [ ] App built locally from source 
  - If so, which branch? (WinUI 3 versions are built off the winui3preview branch)

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/31008169)
